manifests/jenkins-s2i.yaml

```diff
@@ -72,6 +72,9 @@ objects:
             kind: ImageStreamTag
             name: ${JENKINS_S2I_SRC_IMAGESTREAM_NAME}
             namespace: ${JENKINS_S2I_SRC_IMAGESTREAM_NAMESPACE}
+          env:
+            - name: JENKINS_UC_DOWNLOAD
+              value: 'https://mirror.xmission.com/jenkins'
```

Comment:
I think https://archives.jenkins.io/ is the canonical source. Can we try that one instead?

Reply:
I'll give it a try today or tomorrow.

Reply:
I tested this and confirms it downloads the plugins. I updated the commit!